packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/utils.ts

```diff
@@ -66,3 +67,25 @@ export function normalizeExtraEntryPoints(
     return normalizedEntry;
   })
 }
+
+export function getSourceMapDevTool(
+  scriptsSourceMap: boolean,
+  stylesSourceMap: boolean,
+  hiddenSourceMap = false,
+  inlineSourceMap = false,
+): SourceMapDevToolPlugin {
+  const include = [];
+  if (scriptsSourceMap) {
+    include.push(/js$/);
+  }
+
+  if (stylesSourceMap) {
+    include.push(/css$/);
```

Comment:
What about sass/others?

Reply:
This is actually for the emitted modules. So it can never be sass etc...

Reply:
Got it, nvm then.